<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFunctionWidget</class>
 <widget class="QWidget" name="CFunctionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>636</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>top</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(119, 118, 123);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="menuBtn">
        <property name="styleSheet">
         <string notr="true">:hover{background: rgb(36, 31, 49);border:0 px;} 
:!hover{border-style:none;border-radius:0px;}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/imgs/resources/menu-w.png</normaloff>:/imgs/resources/menu-w.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>36</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>905</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="minBtn">
        <property name="styleSheet">
         <string notr="true">:hover{background: rgb(36, 31, 49);border:0 px;} 
:!hover{border-style:none;border-radius:1px;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/imgs/resources/minus-w.png</normaloff>:/imgs/resources/minus-w.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sizeBtn">
        <property name="styleSheet">
         <string notr="true">:hover{background: rgb(36, 31, 49);border:0 px;} 
:!hover{border-style:none;border-radius:1px;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/imgs/resources/qr-code-w.png</normaloff>:/imgs/resources/qr-code-w.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="styleSheet">
         <string notr="true">:hover{background: rgb(224, 27, 36);border:0 px;} 
:!hover{border-style:none;border-radius:1px;}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/imgs/resources/close-w.png</normaloff>:/imgs/resources/close-w.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget2_1" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(36, 31, 49);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0,9">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>47</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="forderBtn">
           <property name="styleSheet">
            <string notr="true">:hover{background: rgb(61, 56, 70);border:0 px;} 
:!hover{border-style:none;border-radius:1px;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource.qrc">
             <normaloff>:/imgs/resources/folder-open-w.png</normaloff>:/imgs/resources/folder-open-w.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moduleBtn">
           <property name="styleSheet">
            <string notr="true">:hover{background: rgb(61, 56, 70);border:0 px;} 
:!hover{border-style:none;border-radius:1px;}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resource.qrc">
             <normaloff>:/imgs/resources/inboxes-w.png</normaloff>:/imgs/resources/inboxes-w.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>445</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="handleWidth">
         <number>0</number>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="widget2_2" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(154, 153, 150);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="widget2_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(94, 92, 100);</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
